client/client/modules/render/tracks/featureCounts/sourcesManager/index.js

```diff
@@ -47,7 +47,7 @@ export default class Index {
         }
         if (!this.colors.hasOwnProperty(source)) {
             const all = Object.values(this.colors);
-            const [best] = array
+            const [,best] = array
```

Comment:
This will skip the best color (i.e. color with minimum amount of sources); instead, we need to skip the first color of `this.colors`